File: BraintreeCore/src/main/java/com/braintreepayments/api/AppSwitchNotAvailableException.java
package com.braintreepayments.api;

/**
 * Error class thrown when app switch to the corresponding wallet is not possible
 */
public class AppSwitchNotAvailableException extends BraintreeException {

    AppSwitchNotAvailableException(String message) {
        super(message);
    }
}

File: BraintreeCore/src/main/java/com/braintreepayments/api/AppSwitchNotAvailableException.kt
package com.braintreepayments.api

import androidx.annotation.RestrictTo

/**
 * Error class thrown when app switch to the corresponding wallet is not possible
 */
open class AppSwitchNotAvailableException @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP) constructor(
    message: String?
) : BraintreeException(message)
